### PR TITLE
chore: Replace favorite icon in NFT with DS icon

### PR DESCRIPTION
**Pull request by brianacnguyen**
## **Description**
- Replace favorite icon in NFT with DS icon

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to wallet
2. Click on NFT
3. Favorite an NFT

### **Before**
Simulator Screenshot - iPhone 15 Pro - 2024-01-11 at 12 21 58

### **After**
296930382-9813f9f0-a8b3-4ab3-ae1e-fe7e74aa2896
Screenshot 2024-01-24 at 4 31 48 PM

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/UI/CollectibleContractElement/index.js [hunk 1]
```diff
@@ -224,7 +226,11 @@ function CollectibleContractElement({
             />
           ) : (
             <View style={styles.favoritesLogoWrapper}>
-              <AntIcons color={importedColors.white} name={'star'} size={24} />
+              <Icon
+                name={IconName.Star}
+                color={IconColor.Alternative}
```
Comment (georgewrmarshall) [inline app/components/UI/CollectibleContractElement/index.js:231]: Should the yellow of the star be changing here is or is it another instance of when a brand color would be appropriate?

**Review by georgewrmarshall — COMMENTED** [review]
Looking good left a comment about the yellow of the star

File: app/components/UI/CollectibleContractElement/index.js (hunk 1 above)
Comment (brianacnguyen) [reply app/components/UI/CollectibleContractElement/index.js:231]: again, this call was made by @Akatori-Design

**Review by georgewrmarshall — DISMISSED** [review]
If the colors have been reviewed by @Akatori-Design and he has the context of how this will be used. I'm ok to approve. I would suggest adding him as a reviewer to this PR

**Comment by Akatori-Design** [comment]
@brianacnguyen Can we actually keep the design and apply `Yellow500`to the yellow instead?
Upon chatting with design, we want to keep that yellow style, and not grey. 
Thanks!
cc: @georgewrmarshall

**Comment by brianacnguyen** [comment]
> @brianacnguyen Can we actually keep the design and apply `Yellow500`to the yellow instead? Upon chatting with design, we want to keep that yellow style, and not grey. Thanks! cc: @georgewrmarshall

Updated

**Review by Akatori-Design — DISMISSED** [review]

**Review by georgewrmarshall — DISMISSED** [review]

File: app/components/UI/SignatureRequest/Root/__snapshots__/Root.test.tsx.snap [hunk 2]
```diff
@@ -44,6 +44,9 @@ exports[`Root should render correctly 1`] = `
           "white": Object {
             "000": "#FFFFFF",
           },
+          "yellow": Object {
+            "500": "#ffd33d",
+          },
```
Comment (georgewrmarshall) [inline app/components/UI/SignatureRequest/Root/__snapshots__/Root.test.tsx.snap:49]: checked value against [design token values ](https://metamask.github.io/design-tokens/?path=/story/colors-brand-colors--figma) ✅ 

Screenshot 2024-01-25 at 8 28 28 AM

File: app/util/theme/temp-tokens/brandColors.ts [hunk 3]
```diff
@@ -15,6 +15,9 @@ export const brandColors: BrandColors = {
   black: {
     '000': '#000000',
   },
+  yellow: {
+    '500': '#ffd33d',
```
Comment (georgewrmarshall) [inline app/util/theme/temp-tokens/brandColors.ts:19]: checked value against [design token values ](https://metamask.github.io/design-tokens/?path=/story/colors-brand-colors--figma) ✅ 

Screenshot 2024-01-25 at 8 28 28 AM

**Review by georgewrmarshall — APPROVED** [review]
LGTM! 🚀 

checked yellow500 value against [design token values ](https://metamask.github.io/design-tokens/?path=/story/colors-brand-colors--figma) ✅ 

Screenshot 2024-01-25 at 8 28 28 AM

**Pull request merged**